backtrader/feeds/yahoo.py

```diff
@@ -108,15 +108,18 @@ def _loadline(self, linetokens):
             h /= adjfactor
             l /= adjfactor
             c = adjustedclose
-            v /= adjfactor
-
+            if v:
+              v /= adjfactor
+            else:
+              v = None
             if self.p.round:
                 decimals = self.p.decimals
                 o = round(o, decimals)
                 h = round(h, decimals)
                 l = round(l, decimals)
                 c = round(c, decimals)
-                v = round(v, decimals)
+                if v:
```

Comment:
If v is `None`, then the operation with `round` would raise a `TypeError` exception (in both Python `2.7` and `3.x`)